lib/gameplay.rb

```diff
@@ -0,0 +1,14 @@
+# rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
+class Gameplay
+  def catch_play(plays)
+    play = play.to_i
+    while plays[play - 1] == 'X' || plays[play - 1] == 'O' || !(play.to_i.positive? && play.to_i <= 9)
+      play = gets.chomp.to_i
+      if plays[play - 1] == 'X' || plays[play - 1] == 'O' || !(play.to_i.positive? && play.to_i <= 9)
+        puts 'Ops, it seems you did not type a number between 1-9 OR someone already choosen this slot before :/'
```

Comment:
:bulb: - [ REQUIRED ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Reply:
:bulb: How to? Instead of using **puts** method here, you can return a specific value. And in the main.rb, you can check the return value of this method to see if it's matching your criteria. If it is matching, then you can use **puts** there, in the main.rb.

Reply:
Issue resolved. Thanks for the review!